pkg/controller/provider/container/ova/model.go

```diff
@@ -62,6 +62,8 @@ type Adapter interface {
 	List(ctx *Context, provider *api.Provider) (itr fb.Iterator, err error)
 	// Get object updates
 	GetUpdates(ctx *Context) (updater []Updater, err error)
+	// Clean unexisting objects within the database
+	DeleteUnexisting(ctx *Context) (updates []Updater, err error)
```

Comment:
we should either changes `updates` to `updater` or to `deletions` (as I commented below)